admin/api_test.go

```diff
@@ -252,3 +253,51 @@ func TestGetState(t *testing.T) {
 		}
 	}
 }
+
+func TestSetConnectors(t *testing.T) {
+	tests := []struct {
+		connectors []connector.ConnectorConfig
+	}{
```

Comment:
The actual method being tested is

```go
func (a *AdminAPI) SetConnectors(connectorConfigs []connector.ConnectorConfig) error {
	return a.connectorConfigRepo.Set(connectorConfigs)
}
```

This can only return database or marshaling errors, so it's actually hard to think of a reasonable failure test case.